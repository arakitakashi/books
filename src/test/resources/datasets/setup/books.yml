books:
  - id: !!com.github.database.rider.core.util.EntityUtils$DeferredValue
    title: "テスト駆動開発"
    author: "Kent Beck"
    publisher: "オーム社"
    price: 3080
  - id: !!com.github.database.rider.core.util.EntityUtils$DeferredValue
    title: "アジャイルサムライ"
    author: "Jonathan Rasmusson"
    publisher: "オーム社"
    price: 2860
  - id: !!com.github.database.rider.core.util.EntityUtils$DeferredValue
    title: "エクストリームプログラミング"
    author: "Kent Beck"
    publisher: "オーム社"
    price: 2420
  - id: !!com.github.database.rider.core.util.EntityUtils$DeferredValue
    title: "Clean Agile"
    author: "Robert C. Martin"
    publisher: "ドワンゴ"
    price: 2640
