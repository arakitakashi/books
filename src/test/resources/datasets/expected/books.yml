books:
  - id: 1  # 仮のID、テストによって異なる可能性があります
    title: "テスト駆動開発"
    author: "Kent Beck"
    publisher: "オーム社"
    price: 3080
  - id: 2  # 仮のID、テストによって異なる可能性があります
    title: "アジャイルサムライ"
    author: "Jonathan Rasmusson"
    publisher: "オーム社"
    price: 2860
  - id: 3  # 仮のID、テストによって異なる可能性があります
    title: "エクストリームプログラミング"
    author: "Kent Beck"
    publisher: "オーム社"
    price: 2420
  - id: 4  # 仮のID、テストによって異なる可能性があります
    title: "Clean Agile"
    author: "Robert C. Martin"
    publisher: "ドワンゴ"
    price: 2640
